.gitignore

```diff
@@ -35,3 +35,7 @@ npm-debug.log
 dev.secret.exs
 
 priv/static/uploads
+
+games/punderplay/zig-cache/
```

Comment:
nit: probably better to put this in a `.gitignore` in the `games/punderplay` directory

Reply:
good call, done!